Guides/UPI/okd4-terraform-openstack/README.md

```diff
@@ -0,0 +1,120 @@
+# Openshift community OKD4 (UPI) on Openstack with terraform
+
+Terraform code to provision OKD4 using user provisioned installer (UPI) and
+glue code to tie it all together
+
+From an openshift installer perspective this is a "bare metal" install:
+https://docs.okd.io/latest/installing/installing_bare_metal/installing-bare-metal.html#installing-bare-metal
+User provided infrastructure is orchestrated using terraform in combination with scripts and an ansible playbook.
+
+Tested on Openstack RDO with Calico as SDN in Openstack and openstack designate
```

Comment:
Interesting, I heard Kuryr would integrate into OpenStack better. Any particular reason to pick Calico?

Reply:
Just that this is what ww have at han, and nrec.no was designed some years back and is used for a lot of other stuff than okd. It is not an option ( at the moment  at least) to swap sdn to something else at the moment. We have to deal with what is there.